README.md

```diff
@@ -661,123 +658,99 @@ my-mbed-os-example (a5ac4bf2e468)
 
 ## Updating programs and libraries
 
-You can update programs and libraries on your local machine so that they pull in changes from the remote sources (GitHub or Mercurial). 
-
-There are two main scenarios when updating:
-
-* Update to a *moving* revision, such as the tip of a branch.
-* Update to a *specific* revision that is identified by a revision hash or tag name.
-
-Each scenario has two cases:
+You can update programs and libraries on your local machine so that they pull in changes from the remote sources (Git or Mercurial). 
 
-* Update with local uncommitted changes - *dirty* update.
-* Update without local uncommitted changes - *clean* update.
+As with any mbed CLI command, `mbed update` uses the current directory as a working context. Before calling `mbed update`, you should change your working directory to the one you want to update. For example, if you're updating mbed-os, use `cd mbed-os` before you begin updating.
 
-As with any mbed CLI command, `mbed update` uses the current directory as a working context, meaning that before calling `mbed update`, you should change your working directory to the one you want to update. For example, if you're updating mbed-os, use `cd mbed-os` before you begin updating.
-
-<span class="tips">**Tip: Synchronizing library references:** Before triggering an update, you may want to synchronize any changes that you've made to the program structure by running ``mbed sync``, which will update the necessary library references and get rid of the invalid ones.</span>
+<span class="tips">**Tip: Synchronizing library references:** Before triggering an update, you may want to synchronize any changes that you've made to the program structure by running ``mbed sync``, which updates the necessary library references and removes the invalid ones.</span>
 
 ### Protection against overwriting local changes
 
-The update command will fail if there are changes in your program or library that `update` could overwrite. This is by design: mbed CLI does not run operations that would result in overwriting local changes that are not yet committed. If you get an error, take care of your local changes (commit or use one of the options below), then rerun `update`.
+The update command fails if there are changes in your program or library that `mbed update` could overwrite. This is by design. mbed CLI does not run operations that would result in overwriting uncommitted local changes. If you get an error, take care of your local changes (commit or use one of the options below), and then rerun `mbed update`.
 
 ### Updating to an upstream version
 
-___Updating a program___
+#### Updating a program
 
-To update your program to another upstream version, go to the root folder of the program and run:
+To update your program to another upstream version, go to the root folder of the program, and run:
```

Comment:
Haha, no. When a coordinating conjunction joins two independent clauses, you must add a comma before the conjunction.